python/bindings/message_decoder.cpp

```diff
@@ -81,22 +100,20 @@ nb::object convert_field(const FieldContainer& field, const PyMessageDatabase::C
     }
 }
 
-PyDecodedMessage::PyDecodedMessage(std::vector<FieldContainer> message_, const oem::MetaDataStruct& meta_, PyMessageDatabase::ConstPtr parent_db_)
-    : fields(std::move(message_)), message_id(meta_.usMessageId), message_crc(meta_.uiMessageCrc), message_name(meta_.acMessageName), time(meta_),
-      measurement_source(meta_.eMeasurementSource), constellation(meta_.constellation), parent_db_(std::move(parent_db_))
-{
-}
+PyMessageBody::PyMessageBody(std::vector<FieldContainer> message_, PyMessageDatabase::ConstPtr parent_db_, std::string name_)
+    : fields(std::move(message_)), parent_db_(std::move(parent_db_)), name(std::move(name_))
```

Comment:
```suggestion
    : name(std::move(name_)), fields(std::move(message_)), parent_db_(std::move(parent_db_))
```
Fixes a compiler warning for fields being initialized out of order.